x-pack/plugins/apm/ftr_e2e/cypress/support/output_command_timings.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+const commands: Array<{
+  name: string;
+  args: string;
+  started: number;
+  endedAt?: number;
+  elapsed: number;
+}> = [];
+
+Cypress.on('test:after:run', (attributes, test) => {
+  /* eslint-disable no-console */
+  console.log(
```

Comment:
Can't we use `cy.log()`?

Reply:
Cypress doesn't support `console.table` which is quite nice for outputting tabular data in an easy to read format.

Reply:
So you've added `console.log` as well to be consistent?